ansible/ec2-docker-playbook.yml

```diff
@@ -0,0 +1,75 @@
+- become: true
+  hosts: all
+  name: Impostor-install
+  gather_facts: true
+  collections:
+    - community.docker
+  vars:
+    docker_image_name: "aeonlucid/impostor"
+    docker_container_name: "impostor"
+  tasks:
+
+  - name: "Impostor/Docker on RHEL-based systems"
+    when: ansible_os_family == "RedHat"
```

Comment:
The AMI I used is an Amazon Linux 2. I don't know if it will show up as RedHat for the family. Can you confirm?

Reply:
Yep, I spun up Amazon Linux 2 on my home vSphere cluster and tested everything on it. Ansible reports it as "RedHat" during the gather_facts stage.

Reply:
That's good news. Thank you. I usually write it so specially to the instance that I don't check those anymore.